plugins/other/pacman

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# DESCRIPTION
+#
+# This plugin monitors the number of pending updates in pacman.
+#
+# WARNINGS
+#
+# This plugin will issue a warning state whenever any package can be updated.
+#
+# ENVIRONMENT
+#
+# env.WARNING The number of updates at which a warning will be issued. If not
```

Comment:
The convention for shell scripts is ALL_CAPS for environment variables, but if Munin prefers lower case then I'll change that.